Bindings/Laerdal.Maui.AzureCommunicationCalling.iOS/Laerdal.Maui.AzureCommunicationCalling.iOS.csproj

```diff
@@ -1,49 +1,48 @@
-﻿ <Project Sdk="Microsoft.NET.Sdk">
-   <PropertyGroup>
-     <PackageId>Laerdal.Maui.AzureCommunicationCalling.iOS</PackageId>
-     <version>2.3.0-beta.5.1</version>
-     <TargetFrameworks>net6.0-ios</TargetFrameworks>
-     <Nullable>enable</Nullable>
-     <ImplicitUsings>true</ImplicitUsings>
-     <IsBindingProject>true</IsBindingProject>
-     <AssemblyName>Laerdal.Maui.AzureCommunicationCalling.iOS</AssemblyName>
-     <Title>AzureCommunicationCalling for net6.0-ios</Title>
-     <Authors>Laerdal Medical</Authors>
-     <PackageReadmeFile>Readme.md</PackageReadmeFile>
-     <Description>
-Binding for Azure Communication Calling SDK for iOS.
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <PackageId>Laerdal.Maui.AzureCommunicationCalling.iOS</PackageId>
+        <version>2.8.0.8</version>
```

Comment:
Could we try to preserve the beta part of the version number?

Reply:
good point,
so you think about something like `2.8.0.8-beta` or maybe something more advanced like full underlying lib number + nuget package build no? 
Example `2.8.0-beta.1.1`, `2.8.0-beta.1.2` ... `2.8.0-beta.1.6`?
That way package consumer will know exactly which native lib is used, and the last number is just for numbering of this nuget package fixes.
What is your opinion?